### DISCUSSION TITLE
实时功能构建索引速度非常慢

### DISCUSSION THREAD

**Discussion by duocairen** [alibaba/havenask#90] [Q&A]
0.3.0版本使用example包里边realtime的case，修改表字段并将分词替换成jieba分词器，其余配置保持不变；往kafka里发送了5000万条数据，半个小时count索引只导入了3万多条数据，机器配置是32核128G，查看机器负载几乎100%空闲，应该修改哪些配置才能加快导入速度呢

**Comment by WenyiDuan** [comment]
先确认下是分词器慢导致还是kafka数据导入问题：1. 使用默认分词器看下导入速率变化 2. 参考[实时功能使用文档](https://github.com/alibaba/havenask/wiki/%E5%AE%9E%E6%97%B6%E5%8A%9F%E8%83%BD%E4%BD%BF%E7%94%A8%E6%96%87%E6%A1%A3)，默认是从topic起始开始导入，如果已经存在很多比全量早的数据，会导入然后过滤掉，可以设置导入起始时间kafka_start_timestamp
